openstack/nova/values.yaml

```diff
@@ -86,13 +90,21 @@ portMetrics: '9102'
 
 sentry_dsn: DEFINE_IN_REGION_CHART
 
+# Use loci only for neutron
+loci:
+  neutron: true
+  nova: false
+
 #TODO we need to move to global or find another way to share image versions
+imageNameNeutron: loci-neutron
+imageVersionNeutronServerM3: null
```

Comment:
There is no `imageVersionNeutronServerM3` anymore

Reply:
ok